src/Tribe/PUE/Checker.php

```diff
@@ -589,14 +593,15 @@ private function get_api_message( $info ) {
 				'</a>'
 			);
 
-			if ( ! empty( $info->api_invalid_message ) ) {
-				$message = wp_kses( $info->api_invalid_message, 'post' );
+			if ( ! empty( $info->api_inline_invalid_message ) ) {
+				$message = wp_kses( $info->api_inline_invalid_message, 'post' );
 			}
 
 			$message = str_replace( '%plugin_name%', $this->get_plugin_name(), $message );
 			$message = str_replace( '%plugin_slug%', $this->get_slug(), $message );
 			$message = str_replace( '%update_url%', $this->get_pue_update_url(), $message );
 			$message = str_replace( '%version%', $info->version, $message );
+			$message = str_replace( '%changelog%', '<a class="thickbox" title="' . $this->get_plugin_name() . '" href="plugin-install.php?tab=plugin-information&plugin=' . $this->get_slug() . '&TB_iframe=true&width=640&height=808">what\'s new</a>', $message );
```

Comment:
... Actually, I take that back, I'm not sure you are right.

The rest of the text is set on theeventscalendar.com and is in English, it is not translatable. If we make "what's new" translatable by itself we'd have the spectre of messages like:

> An update is available! Download now or Quoi de neuf.

Which is fun, but an odd mixture.